client/attest.go

```diff
@@ -32,17 +32,17 @@ type AttestOpts struct {
 	// firmware event log, where PCRs 0-9 and 14 are often measured. If the two
 	// logs overlap, server-side verification using this library may fail.
 	CanonicalEventLog []byte
-	// Indicates whether the AK certificate chain should be retrieved for validation.
-	// If true, Key.Attest() will construct the certificate chain by making GET requests to
-	// the contents of Key.cert.IssuingCertificateURL.
-	FetchCertChain bool
+	// If non-nil, will be used to fetch the AK certificate chain for validation.
+	// Key.Attest() will construct the certificate chain by making GET requests to
+	// the contents of Key.cert.IssuingCertificateURL using this client.
+	FetchCertChainClient *http.Client
```

Comment:
Nit on naming: `CertChainFetcher`? It already has type `Client`, so this name would avoid having the type in the name.  https://g3doc.corp.google.com/go/g3doc/style/decisions.md?cl=head#repetitive-with-type